docs/personas.md

```diff
@@ -1,7 +1,7 @@
 # Knative Personas
 
 When discussing user actions, it is often helpful to
-[define specific user roles](<https://en.wikipedia.org/wiki/Persona_\(user_experience\)>)
+[define specific user roles](<https://en.wikipedia.org/wiki/Persona_(user_experience)>)
```

Comment:
@mattmoor it seems `prettier` and the md link checker don't play well together. Any ideas how to fix it?